js/index.js

```diff
@@ -171,6 +172,14 @@ function renderSearch() {
   }
   search = _.debounce(search, 1000);
 
+  function renderEmoji(code) {
+    return Mustache.render(emojiTemplate, {code: code});
+  }
+
+  function filterEmoji(str) {
+    return str.replace(/\:([a-z_+\-0-9]*)\:/g, renderEmoji('$1'));
```

Comment:
@julmot  I didn't find a best way to filter html tag context only. we should not insert any emoji symbol to html tag attribute.